src/reactor/fns.js

```diff
@@ -330,22 +331,21 @@ export function evaluate(reactorState, keyPathOrGetter) {
   }
 
   // Must be a Getter
-  // if the value is cached for this dispatch cycle, return the cached value
-  if (isCached(reactorState, keyPathOrGetter)) {
-    // Cache hit
-    return evaluateResult(
-      getCachedValue(reactorState, keyPathOrGetter),
-      reactorState
-    )
-  }
 
-  // evaluate dependencies
-  const args = getDeps(keyPathOrGetter).map(dep => evaluate(reactorState, dep).result)
-  const evaluatedValue = getComputeFn(keyPathOrGetter).apply(null, args)
+  const cache = reactorState.get('cache')
+  var cacheEntry = cache.lookup(keyPathOrGetter)
```

Comment:
> do you think cacheLookup here should also handle the resolution of storeStates versus externalizing it?

The caches are generalized right now and don't have nuclear specific logic. I felt like this was still the right place for that logic